v1/remote/image.go

```diff
@@ -90,6 +90,10 @@ func (r *remoteImage) RawManifest() ([]byte, error) {
 	}
 	defer resp.Body.Close()
 
+	if err := checkError(resp); err != nil {
+		return nil, err
+	}
+
 	if resp.StatusCode != http.StatusOK {
 		// TODO(jonjohnsonjr): Parse this into a structured error.
```

Comment:
We should be able to drop this now

Reply:
`checkError` doesn't handle 5xx or unsupported 2xx

Reply:
Done